apps/web/components/examples/blockchain/api-client-demo.tsx

```
@@ -0,0 +1,368 @@
+
+        {/* Performance Metrics */}
+        <div className="bg-white rounded-lg border border-gray-200 p-6">
+          <div className="flex items-center justify-between mb-4">
+            <h2 className="text-lg font-semibold text-gray-900 flex items-center gap-2">
+              <ChartBarIcon className="h-5 w-5" />
+              Performance Metrics
+            </h2>
+            <button
+              onClick={clearCache}
+              className="text-sm text-blue-600 hover:text-blue-700 flex items-center gap-1"
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)